packages/plugin-lightningcss/src/loader.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * modified from https://github.com/fz6m/lightningcss-loader
+ * MIT License https://github.com/fz6m/lightningcss-loader/blob/main/LICENSE
+ * Author @fz6m
+ */
+import type { LoaderContext } from 'webpack';
+import { transform as _transform } from 'lightningcss';
```

Comment:
The `_transform` can be lazy imported, so if user use a custom implementation, we do not need to load two copies of lightningcss.

Reply:
but we `export { lightningcss };` in the index file, so the lightningcss will be imported by default 😂

Reply:
get 👍